tx-indexer/src/config.rs

```diff
@@ -301,6 +341,31 @@ impl ScyllaStorageManager for ScyllaDBManager {
                     VALUES (?, ?)",
             )
             .await?,
+
+            add_transaction_process: Self::prepare_write_query(
```

Comment:
I would suggest renaming to `process_add_transaction` or even something like `cache_add_transaction` and the similar names below for better readability.

Right now, having the `_process` suffix is misleading.